packages/expo-av/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Fix iOS build with Expo SDK 44. [#15661](https://github.com/expo/expo/pull/15661)
```

Comment:
```suggestion
- Fix iOS build with Expo SDK 44 and React Native 0.65+. [#15661](https://github.com/expo/expo/pull/15661)
```

Just a small clarification for the changelog entry. 

It looks like the code adding conflicting variable name on the RN side has landed in `0.65` release:
* https://github.com/facebook/react-native/commit/6f02942dc12fe95a153c2a4cc17ae9bcaa7b0154